tox.ini

```diff
@@ -1,14 +1,14 @@
 [tox]
 envlist =
-    py{35,36,37}-django{111,20,21,22}-{postgresql,sqlite}
+    py{36,37,38}-django{111,20,21,22,30}-{postgresql,sqlite}
```

Comment:
Although we can remove it from the docs, we could leave it in our `tox`. We could add, for example:

```
py35-django{111,20,21,22}-{postgresql,sqlite}
```